pipelines.py

```diff
@@ -14,9 +15,16 @@ def unet(config, train_mode):
 
     loader = preprocessing_generator(config, is_train=train_mode)
     unet = Step(name='unet',
-                transformer=PyTorchUNetStream(**config.unet) if config.execution.stream_mode else PyTorchUNet(
-                    **config.unet),
+                transformer=PyTorchUNetStream(**config.unet)
+                if config.execution.stream_mode
+                else PyTorchUNet(**config.unet),
+                input_data=['input'],
                 input_steps=[loader],
+                adapter={
+                    'datagen': ([(loader.name, 'datagen')]),
+                    'validation_datagen': ([(loader.name, 'validation_datagen')]),
+                    'meta_valid': ([('input', 'meta_valid')])
```

Comment:
@taraspiotr this is a perfect spot to use this multi-key nature of data that is passed to steps. For example one could have 'callback_data''{'meta_valid':data} and 'specs':{'train_mode=True} there.